src/engine/renderer/tr_local.h

```diff
@@ -1085,8 +1085,8 @@ static inline void halfToFloat( const f16vec4_t in, vec4_t out )
 	{
 	  COLLAPSE_none,
 	  COLLAPSE_generic, // used before we know it's another one
-	  COLLAPSE_lighting_PHONG,
-	  COLLAPSE_lighting_PBR,
+	  COLLAPSE_lighting_SPECULAR,
+	  COLLAPSE_lighting_PHYSICAL,
```

Comment:
I don't see the sense of renaming PBR to PHYSICAL. The name PHYSICAL seems pretty meaningless while PBR might mean something to a person who is familiar with rendering techniques.